mscore/shortcut.cpp

```diff
@@ -332,8 +332,8 @@ Shortcut Shortcut::_sc[] = {
          STATE_NORMAL | STATE_NOTE_ENTRY,
          "note-input-timewise",
          QT_TRANSLATE_NOOP("action","Insert"),
+         QT_TRANSLATE_NOOP("action","Note input: Enter Timewise mode"),
```

Comment:
Better "Insert mode"? We got rid of the term 'timewise' quite a while ago

Reply:
done